tests/containers/podman_integration.pm

```diff
@@ -76,6 +76,11 @@ sub run {
     install_packages(@pkgs);
     install_ncat;
 
+    $oci_runtime = get_var("OCI_RUNTIME", script_output("podman info --format '{{ .Host.OCIRuntime.Name }}'"));
```

Comment:
```suggestion
    $oci_runtime = get_var("OCI_RUNTIME", 'crun');
```
I think we can apply the default `crun` here instead of querying podman. `crun` is used AFAIK everywhere as default.

Reply:
`crun` is default on TW but `runc` is default on SLES.  And I want to keep the code clean without making those assumptions.

Reply:
Where is this set then?

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/containers/podman_integration.pm#L34

Reply:
Found it, line 34 (not part of this PR).

Reply:
Ok, makes sense.